geode-management/src/main/java/org/apache/geode/management/internal/ClientClusterManagementService.java

```diff
@@ -133,4 +133,8 @@ private String getEndpoint(CacheElement config) {
     return ((RestfulEndpoint) config).getEndpoint();
   }
 
+  public String ping() {
```

Comment:
Can this rather be `boolean isConnected()`? That way the returned value is more intuitive.